admin/components/Settings/index.js

```diff
@@ -0,0 +1,302 @@
+            placeholder={fieldSiteId.placeholder}
+            value={siteId}
+            onChange={setSiteId}
+          />
+        </FormField>
+        <FormField label={fieldSsrServer.label}>
+          <StyledTextInput
+            status={ssrServerStatus}
+            placeholder={fieldSsrServer.placeholder}
+            value={ssrServer}
+            onChange={setSsrServer}
+          />
+        </FormField>
+        <FormField label={fieldStaticServer.label}>
+          <StyledTextInput
+            status={staticServerStatus}
+            placeholder={fieldStaticServer.placeholder}
+            value={staticServer}
+            onChange={setStaticServer}
+          />
+        </FormField>
+        <FormField label={fieldAmpServer.label}>
+          <StyledTextInput
+            status={ampServerStatus}
+            placeholder={fieldAmpServer.placeholder}
+            value={ampServer}
+            onChange={setAmpServer}
+          />
+        </FormField>
+        <Box
+          direction="row"
+          justify="between"
+          align="start"
+          margin={{ bottom: "18px" }}
+        >
+          <Box direction="row" justify="between" align="center" width="262px">
+            <Paragraph margin={{ vertical: "0", left: "12px", right: "20px" }}>
+              {fieldForceFrontpage.label}
+            </Paragraph>
+            <CheckBox
+              toggle
+              checked={frontpageForced}
+              onChange={setFrontpageForced}
+            />
+          </Box>
+          <Comment margin={{ vertical: "0" }}>
+            {fieldForceFrontpage.comment}
+          </Comment>
+        </Box>
+        <Box
+          direction="row"
+          justify="between"
+          align="center"
+          margin={{ bottom: "18px" }}
+        >
+          <Box direction="row" justify="between" align="center" width="262px">
+            <Paragraph margin={{ vertical: "0", left: "12px", right: "20px" }}>
+              {fieldHtmlPurifier.label}
+            </Paragraph>
+            <CheckBox
+              toggle
+              checked={htmlPurifierActive}
+              onChange={setHtmlPurifierActive}
+            />
+          </Box>
+          <Button
+            label={fieldHtmlPurifier.button}
+            onClick={purgeHtmlPurifierCache}
+          />
+        </Box>
+        <FormField label={fieldExcludes.label}>
+          <TextArea
+            placeholder={fieldExcludes.placeholder}
+            value={excludes}
+            onChange={setExcludes}
+          />
+        </FormField>
+        <FormField label={fieldApiFilters.label}>
+          <TextArea
+            placeholder={fieldApiFilters.placeholder}
+            value={apiFilters}
+            onChange={setApiFilters}
+          />
+        </FormField>
+      </Form>
+    </Options>
+    <Button
+      primary
+      margin={{ left: "auto" }}
+      type="submit"
+      label={saveButtonText}
+      onClick={saveSettings}
```

Comment:
You can also use a proper `<form>` here, as shown in https://github.com/frontity/wp-plugin/pull/73/commits/53fd3f061600fdba528c07f41437ee15553e0b5e